static/src/js/util/accessMethods/__tests__/buildAccessMethods.test.js

```diff
@@ -267,86 +263,98 @@ describe('buildAccessMethods', () => {
     }
     const isOpenSearch = false
 
-    const methods = buildAccessMethods(collectionMetadata, isOpenSearch)
+    buildAccessMethods(collectionMetadata, isOpenSearch)
 
-    expect(methods).toEqual({
-      harmony0: {
-        enableTemporalSubsetting: true,
-        enableSpatialSubsetting: true,
-        hierarchyMappings: [
-          {
-            id: 'V100000-EDSC'
-          },
-          {
-            id: 'V100001-EDSC'
-          },
-          {
-            id: 'V100002-EDSC'
-          },
-          {
-            id: 'V100003-EDSC'
-          }
-        ],
-        id: 'S100000-EDSC',
-        isValid: true,
-        keywordMappings: [],
+    expect(buildHarmonyMock).toBeCalledTimes(1)
+
+    expect(buildHarmonyMock).toHaveBeenNthCalledWith(
+      1,
+      {
+        conceptId: 'S100000-EDSC',
         longName: 'Mock Service Name',
         name: 'mock-name',
-        supportedOutputFormats: [
-          'GEOTIFF',
-          'PNG',
-          'TIFF',
-          'NETCDF-4'
-        ],
-        supportedOutputProjections: [],
-        supportsBoundingBoxSubsetting: true,
-        supportsShapefileSubsetting: false,
-        supportsTemporalSubsetting: false,
-        supportsVariableSubsetting: true,
-        supportsConcatenation: true,
-        defaultConcatenation: true,
-        enableConcatenateDownload: true,
+        serviceOptions: {
+          aggregation: {
+            concatenate: {
+              concatenateDefault: true
+            }
+          },
+          interpolationTypes: ['Bilinear Interpolation', 'Nearest Neighbor'],
+          subset: {
+            spatialSubset: { boundingBox: { allowMultipleValues: false } },
+            variableSubset: { allowMultipleValues: true }
+          },
+          supportedOutputProjections: [{ projectionName: 'Polar Stereographic' }, { projectionName: 'Geographic' }],
+          supportedReformattings: [{
+            supportedInputFormat: 'NETCDF-4',
+            supportedOutputFormats: ['GEOTIFF', 'PNG', 'TIFF', 'NETCDF-4']
```

Comment:
The formatting on this whole section is off compared to what was there originally. These arrays and objects keys/values should all be on newlines. Please try and follow the formatting that was here.